test/functional/loopTestDriver.py

```diff
@@ -18,6 +18,10 @@ class LoopTestDriver():
     def setUp(self, marionette):
         self.marionette = marionette
 
+    def set_context(self, context):
+        self.context = context
+        self.marionette.set_context(context)
```

Comment:
It might be nice to file a bug on Marionette requesting get_context() so that this is unnecessary.